E-Commerce/readme.md

```diff
@@ -45,13 +45,45 @@ erDiagram
 
 Write SQL queries to complete the following tasks:
 
-- [ ] List all the products whose name contains the word "socks"
-- [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
-- [ ] List the 5 most expensive products
-- [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
-- [ ] List all orders, including order items, from customer named Hope Crosby
-- [ ] List all the products in the order ORD006. The result should only contain the columns product_name, unit_price, and quantity
-- [ ] List all the products with their supplier for all orders of all customers. The result should only contain the columns name (from customer), order_reference, order_date, product_name, supplier_name, and quantity
+- [*] List all the products whose name contains the word "socks"
+SELECT * FROM products WHERE product_name LIKE ‘%socks%’;
+
+- [*] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+SELECT prod_id, product_name, unit_price, supp_id
+FROM product_availability pa INNER JOIN products p on (pa.prod_id=p.id)
+INNER JOIN suppliers s on (pa.supp_id=s.id)
+WHERE unit_price > 100;
```

Comment:
CORRECT - Whilst this is not wrong, there is no need to join in the suppliers table, as you are only returning no details from it (only the supp_id which is in product_availability)